jaxrs-tck/pom.xml

```diff
@@ -96,5 +110,17 @@
             <artifactId>hamcrest-library</artifactId>
             <version>[2.2, 2.3-A00)</version>
         </dependency>
+
+        <dependency>
+            <groupId>org.jboss.arquillian.junit5</groupId>
+            <artifactId>arquillian-junit5-container</artifactId>
+            <version>${arquillian-bom.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.squareup.okhttp3</groupId>
```

Comment:
We might need a "uses" CQ for this. What is it doing?

Reply:
It's just a simple HTTP client.

> Good starting point, but how to see this code in action? Can you put an example ontop that is actually executable using `mvn verify`? Thanks! :-)

I have this configured to run with Open Liberty and GlassFish (depending on profile, `-Parq-liberty-managed` or `-Parq-glassfish-managed`)  at https://github.com/andymc12/jakartaRest31TCK-openliberty 

The problem is that although the test framework claims that the tests run and pass, they aren't actually executed... I'm investigating this issue further with some folks over slack at: https://eclipsefoundationhq.slack.com/archives/C0131MLD538/p1624542758005000

If we don't make progress soon, I may give TestNG a try.

Reply:
As Andy replaces OKHTTP3 by java.net.HttpClient, I am closing this thread.